docs/tools.rst

```diff
@@ -55,10 +58,43 @@ Example
 .. code-block:: bash
 
    nvidiaAIAAListModels \
-      -server http://10.110.45.66:5000/v1 \
+      -server http://0.0.0.0:5000 \
       -label spleen
 
 
+Session
+------------
+
+Provides implementation for ``nvidia::aiaa::Client::create_session()`` API.
```

Comment:
function is called "createSession" in cpp,
but this tool also provide implementation of "getSession" function
what do we want

Reply:
fixed